File: app/src/main/java/com/karim/booksapp/data/models/pojos/Item.java

package com.karim.booksapp.data.models.pojos;

import com.squareup.moshi.Json;

public class Item {

    @Json(name = "kind")
    public String kind;
    @Json(name = "id")
    public String id;
    @Json(name = "etag")
    public String etag;
    @Json(name = "selfLink")
    public String selfLink;
    @Json(name = "volumeInfo")
    public VolumeInfo volumeInfo;
    @Json(name = "saleInfo")
    public SaleInfo saleInfo;
    @Json(name = "accessInfo")
    public AccessInfo accessInfo;
    @Json(name = "searchInfo")
    public SearchInfo searchInfo;

}

File: app/src/main/java/com/karim/booksapp/models/pojos/Item.kt
package com.karim.booksapp.models.pojos

import com.squareup.moshi.Json

class Item {

    @Json(name = "kind")
    var kind: String? = null
    @Json(name = "id")
    var id: String? = null
    @Json(name = "etag")
    var etag: String? = null
    @Json(name = "selfLink")
    var selfLink: String? = null
    @Json(name = "volumeInfo")
    var volumeInfo: VolumeInfo? = null
    @Json(name = "saleInfo")
    var saleInfo: SaleInfo? = null
    @Json(name = "accessInfo")
    var accessInfo: AccessInfo? = null
    @Json(name = "searchInfo")
    var searchInfo: SearchInfo? = null

}
